### PR TITLE
Check mtu

### PR DESCRIPTION
**Pull request by mikalstill**

### CODE REVIEW

File: ansible/deploy.yml [hunk 1]
```diff
@@ -120,6 +120,32 @@
...
+    - name: Extract default interface MTU
+      set_fact:
+        node_mtu: "{{node_mtu_complex.stdout}}"
+
+    - name: Log node MTU
+      debug:
+        msg: "Node MTU is {{node_mtu}}"
+
+    - name: Abort if default interface MTU is too low
+      shell: exit 1
```
Comment (mandoonandy) [inline ansible/deploy.yml]: Sorry to annoy: 
Would a "fail" be better here. You will get a pretty red message using a fail.

**Review by mandoonandy — CHANGES_REQUESTED** [review]

**Comment by mikalstill** [comment]
Ready for another review.

**Review by mandoonandy — APPROVED** [review]
LGTM

**Comment by mandoonandy** [comment]
No label and no linked issue.
Should this be in v0.2.2?

**Comment by mikalstill** [comment]
Yes, this should be in v0.2.2. I'll create an issue, label it, and then do the cherry pick (that's how I tracked the other example of something released into v0.2-devel without an issue). That is, I think _everything_ which lands in v0.2 should have an issue for the purposes of tracking.

**Pull request merged**